datamodel/initdb.d/06_metadata.sql

```diff
@@ -65,4 +65,14 @@ ALTER TABLE dcsa_im_v3_0.transport_document
 ALTER TABLE dcsa_im_v3_0.shipment
     ADD COLUMN IF NOT EXISTS updated_date_time timestamp with time zone NOT NULL;
 
+DROP TABLE IF EXISTS dcsa_im_v3_0.ebl_solution_provider_type CASCADE;
```

Comment:
Wrong file - this should go into `05_reference_implementation_support.sql`

Reply:
Fixed!